src/Client/ConcordiumClientOptions.cs

```diff
@@ -0,0 +1,36 @@
+
+using Grpc.Net.Client;
+
+#pragma warning disable CS8618
+
+namespace Concordium.Sdk.Client;
+
+/// <summary>
+/// Configurations to Concordium Client.
+/// </summary>
+public sealed class ConcordiumClientOptions
+{
+    /// <summary>
+    /// Endpoint of a resource where the V2 API is served.
+    /// </summary>
+    public Uri Endpoint { get; init; }
```

Comment:
I have checked this and the compiler does not prevent the null pointer, which I think is a bad user experience.

Reply:
I have now added a default value which I think is the best approach. It is written in the header docs.

Reply:
I don't think a default endpoint is right for the SDK, since this will vary a lot depending on your project.
I also think this is a bad user experience, since a user without a node running with gRPC enabled on this specific port will see a gRPC error with status "Unavailable" and "connection refused", which is expected in this case but not helpful for someone who forgot to change the endpoint.
Also, none of the other SDKs behaves like this.

Reply:
I think we need to force the user to provide an endpoint in the API, one way would be adding it as a parameter for the client constructors

Reply:
I have removed it from the configurations should that it defaults to null when loaded in as configurations. Then a validation is done in the client.